src/combobox/index.ts

```diff
@@ -48,14 +48,14 @@ import { customElement } from '@dojo/widget-core/decorators/customElement';
 export interface ComboBoxProperties extends ThemedProperties, LabeledProperties {
 	clearable?: boolean;
 	disabled?: boolean;
-	getResultLabel?(result: any): string;
+	getResultLabel?(result: any): DNode;
 	getResultSelected?(result: any): boolean;
 	widgetId?: string;
 	inputProperties?: TextInputProperties;
 	invalid?: boolean;
 	isResultDisabled?(result: any): boolean;
 	onBlur?(value: string, key?: string | number): void;
-	onChange?(value: string, key?: string | number): void;
+	onChange?(value: DNode, key?: string | number): void;
```

Comment:
is this right? all the other `on{thing}` callbacks return the value as a string

Reply:
@tomdye the other `on{thing}` callbacks use the `value` prop, while `onChange` is fired when changing options. It would probably originally have been better to do `onChange?(result: any, index: number, key?: string | number)` like `Listbox` does, but that would be a breaking change 😝

Reply:
The other, possibly easier, option is to add `getResultValue` and return that in `onChange`. That might be better anyway, since `onChange` now is used for both typing in the `TextInput` and selecting a result.